### DISCUSSION TITLE
How to monitor KEDA with Prometheus?

### DISCUSSION THREAD

**Discussion by ajames-branch** [kedacore/keda#3605] [General] [answered]
Is it possible to monitor KEDA with Prometheus?
If so, could you please guide me with relevant info/links?

**Comment by tomkerkhove** [comment, marked as answer]
These are the metrics that we have today: https://keda.sh/docs/latest/operate/prometheus/

**Reply by ajames-branch** [reply]
Thank you @tomkerkhove. This is helpful, let me check it out.

**Reply by QuentinBisson** [reply]
@ajames-branch did you manage to create a curated set of alerts ? Could you share them ?
